src/test/java/org/folio/rest/api/LegacyItemEffectiveLocationMigrationScriptTest.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.rest.api;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+import org.junit.Test;
+
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+import io.vertx.sqlclient.Row;
+import io.vertx.sqlclient.RowSet;
+
+public class LegacyItemEffectiveLocationMigrationScriptTest extends MigrationTestBase {
+
+  private static final String CREATE_FUNCTIONS_SCRIPT = loadScript("effectiveLocationFunctionCreator.sql");
+  private static final String DROP_EFFECTIVE_LOCATION_FUNCTION_SCRIPT = loadScript("dropLegacyItemEffectiveLocationFunctions.sql");
+
+  @Test
+  public void canDropLegacyEffectiveItemLocationFunctions() throws Exception {
```

Comment:
If lombok is part of the module, this could use `@SneakyThrows` (which preserves the specific types whilst also removing noise) and get rid of the `throws Exception`